.github/workflows/db_updates.yml

```diff
@@ -0,0 +1,52 @@
+name: CI Pipeline
+
+on:
+  schedule:
+    - cron: "0 8 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install poetry
+        poetry version "$(poetry version -s).$PATCH_NUMBER"
+        poetry install -v
+      env:
+        PATCH_NUMBER: ${{ github.run_number }}
+    - name: Build
+      run: |
+        poetry build
+    - name: Lint
+      run: |
+        poetry run poe lint
+    - name: Test
+      run: |
+        poetry run poe unit_test
+    - name: Restore database 
+      uses: actions/cache/restore@v4
+      with:
+        key: ${{ github.head_ref }}.${{ github.sha }.db}
+        path: |
+          ~/.vulndb/vuln.db
+    - name: Build database
+      if: steps.restore-database.outputs.cache-hit
+      run: |
```

Comment:
If this is restored from cache, then we will presumably have a problem trying to create it?

Reply:
yes, removed the create line, it was left in by mistake